team.rst

```diff
@@ -0,0 +1,192 @@
+     </div>
+     <a href="https://github.com/emmanuelle" class="team-member-name">Emmanuelle Gouillart</a>
+     <div class="team-member-handle">@emmanuelle</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars2.githubusercontent.com/u/6528957?u=2a7658d9dbc4af127f9d90003462e37de4f73fba&v=4&s=40"/>
+     </div>
+     <a href="https://github.com/grlee77" class="team-member-name">Gregory R. Lee</a>
+     <div class="team-member-handle">@grlee77</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars2.githubusercontent.com/u/90008?u=19cf915f2609e5b272a4144d8a2840cc6e51f28a&v=4&s=40"/>
+     </div>
+     <a href="https://github.com/hmaarrfk" class="team-member-name">Mark Harfouche</a>
+     <div class="team-member-handle">@hmaarrfk</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars0.githubusercontent.com/u/2184487?u=09e9b57497b6f07a61ee788d722fa9e6988f5c17&v=4&s=40"/>
+     </div>
+     <a href="https://github.com/JDWarner" class="team-member-name">Josh Warner</a>
+     <div class="team-member-handle">@JDWarner</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars3.githubusercontent.com/u/492549?v=4&s=40"/>
+     </div>
+     <a href="https://github.com/jni" class="team-member-name">Juan Nunez-Iglesias</a>
+     <div class="team-member-handle">@jni</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars0.githubusercontent.com/u/20140352?u=aa80243f7c2da2341ac944d70a33daef0c369aed&v=4&s=40"/>
+     </div>
+     <a href="https://github.com/lagru" class="team-member-name">Lars Grüter</a>
+     <div class="team-member-handle">@lagru</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars2.githubusercontent.com/u/3438227?u=dd1bcfe9172643955985de8ecdf7ba0627725557&v=4&s=40"/>
+     </div>
+     <a href="https://github.com/rfezzani" class="team-member-name">Riadh Fezzani</a>
+     <div class="team-member-handle">@rfezzani</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars3.githubusercontent.com/u/335370?u=b9359e95f23c1864c804ba22f41bcf540951b20e&v=4&s=40"/>
+     </div>
+     <a href="https://github.com/sciunto" class="team-member-name">François Boulogne</a>
+     <div class="team-member-handle">@sciunto</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars1.githubusercontent.com/u/1315589?u=925c7a474919699fc03c39a75876dd0abc0fa790&v=4&s=40"/>
```

Comment:
I would prefer to have a photo 😅 . The one from the Google Scholar profile would work - https://scholar.googleusercontent.com/citations?view_op=medium_photo&user=G4gN8BEAAAAJ&citpid=1 .

Reply:
do you mind using it on Github? It would be much easier for generating the gallery.

Reply:
@soupault We just use the one you have on GitHub.  Could you update your GitHub profile picture?

Reply:
I guess I don't mind 🙂 ... It is updated now.